### ISSUE TITLE
Unable to put result on the clipboard.

### ISSUE THREAD

**Issue by IndrajeetPatil** [tidyverse/reprex#173]
Hi,

I have been using `reprex` almost on a daily basis for a month now. But ever since I installed the new version of reprex, it has stopped working properly and gives me the following error-
```
> reprex::reprex(input = "so_answer.R", venue = "so")
Unable to put result on the clipboard. How to get it:
  * Capture what reprex() returns.
  * Use `outfile = "foo"` to request output in specific file.
  * See the temp file:
    - C:/Users/Indrajeet Patil/AppData/Local/Temp/Rtmp6LPKDJ/file23a428855d1_reprex.md
```
I reinstalled `reprex`, `clipr`, `knitr`, but still the same issue persists. Does this have to do with some other package I might have installed that is interfering with its functionality?

**Comment by cderv** [comment]
There is some issue with the version currently on CRAN (version 0.1.2). 
CRAN policy changed regarding clipboard, changes have been made in different packages but not all are on the CRAN right now. You can refer to #171 for further information.

Right now, you should installed from github to have the last version working.

**Comment by IndrajeetPatil** [comment]
Oh, I see. Thanks! I will install the package from GitHub from henceforth.

**Comment by GillesSanMartin** [comment]
Hello

I had the same problem reported here with the CRAN version. 

I installed from GitHub (`reprex_0.2.0`) and I had the following message (when runing reprex from Rstudio in Ubuntu) even if I was providing an expression instead of using the clipboard

```
Rendering reprex...
 
Erreur : path for html_dependency not found: 
```

I tried the same with R running in a console instead of Rstudio.   
I received the following message `Clipboard on X11 requires 'xclip' (recommended) or 'xsel'`

After installing `xclip`, it worked from the console and now it works also from Rstudio after restarting it.  
 I don't know exactly what solved what...

Thanks for the nice package by the way...
